pkg/loadbalancers/l4netlb.go

```diff
@@ -70,8 +70,8 @@ type L4NetLB struct {
 	enableStrongSessionAffinity bool
 	networkInfo                 network.NetworkInfo
 	networkResolver             network.Resolver
-
-	svcLogger klog.Logger
+	enableWeightedLB            bool
```

Comment:
nit: it is already a part of lb struct, also other enableSmth don't have it
```suggestion
	enableWeighted            bool
```